frontend/src/types/form.ts

```diff
@@ -1,6 +1,14 @@
 import type { Chain, Token } from './data';
 
 export type RequestFormResult = {
+  amount: string;
```

Comment:
I tried to move them to the InputComponents but I couldn't import the typing in the composables. .ts seem not to be able to import from .vue files.

Reply:
Ah, because of the `SelectorOption`? Okay, anyway. Let's don't block this PR because of that and see if we can resolve this when you work on the custom selector component.